portfolio/src/main/webapp/index.html

```diff
@@ -30,6 +30,13 @@ <h1 style="font-family:verdana; color: whitesmoke">Ferrin Threatt's Portfolio</h
       </button>
       <p style="font-family:verdana; color:whitesmoke" id="myself-container"></p>
       <p style="font-family:verdana; color:whitesmoke">Click <a href="/myself">here</a> to see the server response directly.</p>
+      <form action="/form-handler" method="POST">
+        <p p style="font-family:verdana; color:rgb(190, 35, 35)">Contact me</p>
```

Comment:
It looks like you might have a typo here and on the next line, "p p" and "pp" instead of just "p". Be sure to fix the closing tags `</p>` as well.